mmcli: add page (#5876)
tldr-pages/tldr: pages/linux/mmcli.md

# mmcli

> Control and monitor the ModemManager.

> > More information: <https://www.freedesktop.org/software/ModemManager/man/1.0.0/mmcli.8.html>.

Reviewer: Refer to `latest` instead of `1.0.0` ?
```suggestion
> More information: <https://www.freedesktop.org/software/ModemManager/man/latest/mmcli.1.html>.
```

Reviewer 2: I tried using "latest", but I didn't think to chanege the section number like that—smart!

- List SMS messages available on the modem:

> `sudo mmcli -m {{modem}} --messaging-list-sms`

Reviewer 3: We prefer long arguments when there are available, checking the man page we can use:
```suggestion
`sudo mmcli --modem={{modem}} --messaging-list-sms`
```

> - Delete a message from the modem by path:

Reviewer 4: ```suggestion
- Delete a message from the modem specifying its path:
```
This wording is more common among the other pages.

Author: I didn't think about that. I searched a bit through the commands to confirm, but I think most tend _not_ to acknowledge their parameters at all. Could it be worth dropping that entirely?

```suggestion
- Delete a message from the modem:
```

![image](https://user-images.githubusercontent.com/22801583/116916190-49781d80-ac4d-11eb-96ba-f7a9fc2122b4.png)
> It seems most of the description that explicitly say "specify the path" do so because the command is setting an output path or similar.
 
![image](https://user-images.githubusercontent.com/22801583/116917124-8a246680-ac4e-11eb-9949-a90f35054ced.png)
> While some mention "specifying its path" or something similar, it's not very common.

![image](https://user-images.githubusercontent.com/22801583/116917543-12a30700-ac4f-11eb-8ae1-853bbf722e92.png)
> Looking at commands like 7z, adb, etc most don't even acknowledge the paramaters, or that a path is required, it's only inferred by the command itself.

Reviewer 4: I wouldn't remove it entirely since we should try to keep the descriptions as simple as possible. I just think that `by path` sounds a little odd. However, I'm not an native speaker, so let's see what @bl-ue thinks.

Reviewer 2: It's actually perfectly normal to say something like that.  I'd prefer a slight more verbose version, however:
```suggestion
- Delete a message from the modem by its path:
```
wdyt @SethFalco?

Author: Objectively speaking, I think @marchersimon has a point with "sounds a little odd". Even if mine's grammatically correct, it could be worth being more verbose if it aids in accessibility. (Baring in mind a lot of non-natives will be using the English version anyway.)

As a native speaker, I prefer my version, but ofc I'd say that, I wrote it. 🥇
I'm pretty comfortable with the terse versions, but think it could be worth using the initial suggestion if the explicitness makes things clearer.

> `sudo mmcli -m {{modem}} --messaging-delete-sms={{path}}`

Reviewer 3: Based on our style guide we prefer `path/to/file`. Could this `path` be `path/to/message_file`? or is it a directory?
```suggestion
`sudo mmcli --modem={{modem}} --messaging-delete-sms={{path}}`
```

Author: I believe it can be, I think its files only so it's a fair suggestion.
Looking at the docs, I can see it accepts an index as well which I didn't realize, not 100% sure if I should document both or just leave it at one for simplicity.

For now, I'll leave it at path only since it's more explicit though, feel free to give your opinion!

Reviewer 4: Even if it is a directory, we could still use `path/to/directory`. Or `path/to/file_or_directory` if both are possible.
